File: 100-Days-Series/Day 37/RemoveSpace.c
#include<stdio.h>
#include<string.h>
// Function for remove spaces from a string
void getRemoveSpaces(char *str){
    int j=0;
    for(int i=0;str[i]!='\0';i++){
        if(str[i]!=' '){
            str[j++]=str[i];
        }
    }
    str[j]='\0';// Null terminate the modified string;
}
// Let's Start here 
void main(){
    char str[100];
    printf("\nEnter a string : ");
    fgets(str,sizeof(str),stdin);
    printf("\nString with Spaces : %s",str);
    str[strcspn(str,"\n")]='\0';// Remove the trailing newline
    getRemoveSpaces(str);
    printf("\nString without Spaces : %s",str);
}

File: 100-Days-Series/Day 37/RemoveSpace.js


// Arrow Function to remove spaces from a string 
const getRemoveSpace=(str)=>{
    temp="";
    for(let i=0;i<str.length;i++){
        ch=str.charAt(i);
        if(ch!=' ')
            temp+=ch;
    }
    return temp;
}
// Let's Start here
str="Full Stack Developer";
console.log("String with spaces : "+str);
console.log("String without spaces : "+getRemoveSpace(str));